lib/permutations.rb

```diff
@@ -1,4 +1,21 @@
 
 def permutations?(string1, string2)
```

Comment:
This doesn't quite work, for example it would fail for `heelo` and `hello`.  See my note below.